twoaxistracking/twoaxistrackerfield.py

```diff
@@ -167,7 +167,7 @@ def get_shaded_fraction(self, solar_elevation,  solar_azimuth,
                 relative_slope=self.relative_slope,
                 slope_azimuth=self.slope_azimuth,
                 slope_tilt=self.slope_tilt,
-                plot=False)
+                plot=plot)
```

Comment:
👍

Reply:
The dangers of not testing plots more extensively..